sample/kotlin/src/main/kotlin/com/datadog/android/sample/traces/TracesViewModel.kt

```diff
@@ -368,14 +368,20 @@ internal class TracesViewModel(
             withinSpan("AsyncOperation", activeSpanInMainThread) {
                 logger.v("Starting Async Operation...")
 
-                for (i in 0..100) {
+                val count = (Random().nextInt() % 50) + 50
+                log("Async op loops $count times")
+                var actualCount = 0
+
+                for (i in 0 until count) {
                     if (isCancelled) {
+                        log("Async operation cancelled")
```

Comment:
minor/ do we want these log statements?

Reply:
Those are logs on the Span object, so they're here to test that the Span.logs are tied to the right Span in the UI